### ISSUE TITLE
Search in Associate searches only by name, not by serial or inventory numbers

### ISSUE THREAD

**Issue by aqvila2k18** [glpi-project/glpi#4367]
* Steps to reproduce (which actions have you made) :
in the components of the computer, I select the item that I want to associate to another computer;
Actions -> Associate -> Computer
in the search field I write the computer name or inventory number or serial number

* Expected result :
search should occur on any of the fields: Name, Serial Number, Inventory/Asset Number
like here
![image](https://user-images.githubusercontent.com/36851646/42935423-42cba782-8b52-11e8-9a05-243afe7bd96b.png)

* Actual result :
search is only by name
like here
![image](https://user-images.githubusercontent.com/36851646/42935673-e8794806-8b52-11e8-94a6-7c3b878adbef.png)

* URL of the page :
https://glpi/front/computer.form.php?id=1524#modal_massaction_contenteba11c89b6b9644493672faada2d12d2

* Screenshot of the problem (if pertinent) :
![image](https://user-images.githubusercontent.com/36851646/42935748-22fa5830-8b53-11e8-85e3-42b3b953bcfa.png)

* Your GLPI setup (you can find it in Setup > General menu, System tab) :
``` 
GLPI 9.3.0 ( => /var/www/glpi)
Installation mode: TARBALL

Server
 
Operating system: Linux glpi 4.4.0-130-generic #156-Ubuntu SMP Thu Jun 14 08:53:28 UTC 2018 x86_64
PHP 7.0.30-0ubuntu0.16.04.1 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc,
	apcu, calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, json, ldap, libxml, mbstring,
	mcrypt, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, readline, session, shmop, sockets, standard, sysvmsg, sysvsem,
	sysvshm, tokenizer, wddx, xml, xmlreader, xmlwriter, xsl, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="10M" safe_mode="" session.save_handler="files"
	upload_max_filesize="10M" 
Software: Apache/2.4.18 (Ubuntu) (Apache/2.4.18 (Ubuntu) Server at glpi Port 443)
	Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36
Server Software: (Ubuntu)
	Server Version: 5.7.22-0ubuntu0.16.04.1
	Server SQL Mode: 
	Parameters: glpi@localhost/glpi
	Host info: Localhost via UNIX socket
	
￼mysqli extension is installed
￼ctype extension is installed
￼fileinfo extension is installed
￼json extension is installed
￼mbstring extension is installed
￼zlib extension is installed
￼curl extension is installed
￼gd extension is installed
￼simplexml extension is installed
￼xml extension is installed
￼ldap extension is installed
￼Zend OPcache extension is installed
￼APCu extension is installed
￼imap extension is not present
￼xmlrpc extension is not present
￼CAS extension is not present
￼Database version seems correct (5.7.22) - Perfect!
￼/var/www/glpi/config : OK
￼/var/www/glpi/files : OK
￼/var/www/glpi/files/_dumps : OK
￼/var/www/glpi/files/_sessions : OK
￼/var/www/glpi/files/_cron : OK
￼/var/www/glpi/files/_graphs : OK
￼/var/www/glpi/files/_lock : OK
￼/var/www/glpi/files/_plugins : OK
￼/var/www/glpi/files/_tmp : OK
￼/var/www/glpi/files/_cache : OK
￼/var/www/glpi/files/_rss : OK
￼/var/www/glpi/files/_uploads : OK
￼/var/www/glpi/files/_pictures : OK
￼/var/www/glpi/files/_log : OK
￼Web access to the files directory should not be allowed but this cannot be checked automatically on this instance.
Make sure acces to error log file is forbidden; otherwise review .htaccess file and web server configuration.
Libraries
 
htmLawed version 1.2.4 in (/var/www/glpi/lib/htmlawed)
SimplePie version 1.5.1 in (/var/www/glpi/vendor/simplepie/simplepie/library)
TCPDF version 6.2.17 in (/var/www/glpi/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/glpi/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/glpi/vendor/true/punycode/src)
iacaml/autolink in (/var/www/glpi/vendor/iamcal/lib_autolink)
sabre/vobject in (/var/www/glpi/vendor/sabre/vobject/lib)

SQL replicas
 
Not active

Notifications
 
Way of sending emails: PHP

Mails receivers
 

Plugins list
 
	fields               Name: Additionnal fields             Version: 1.8.1      State: Enabled
	datainjection        Name: File injection                 Version: 2.5.2      State: Enabled
	geninventorynumber   Name: Inventory number generation    Version: 2.2.0      State: Enabled
	uninstall            Name: Item's uninstallation          Version: 2.4.1      State: Enabled
	genericobject        Name: Objects management             Version: 2.6.0      State: Enabled
	reports              Name: Reports                        Version: 1.11.3     State: Enabled

```

**Comment by trasher** [comment]
>   search should occur on any of the fields: Name, Serial Number, Inventory/Asset Number
like here

Where did you see that one?

**Comment by wacko0jacko** [comment]
I use 9.3.4 and I have the same issue in the 'Projects' section. When I'm trying to assign the asset to the project (Projects > Items > Add).

**Comment by cedric-anne** [comment]
Fixed by #6091
